### DISCUSSION TITLE
Export all settings?

### DISCUSSION THREAD

**Discussion by NormPlum** [portmaster-community/apps#11] [Q&A]
Is there a way to export all settings from Portmaster? I know I can export settings from different section (Global settings, App settings, etc.), but not only are these individual sections, you also have to manually tick the boxes next to each setting you want to export...

I'd like a way to export all non-default settings, from global ones to app-specific ones. Is this currently possible?

**Comment by NormPlum** [comment]
Not sure this was the right place to ask this, so made a feature request here: https://github.com/safing/portmaster/issues/1492

**Reply by Raphty** [reply]
there is not, we will ad a sync feature in the near future which will basically a full export to a folder.
This is considered a convenience feature though and will therefore be included in Plus and Pro. 
This might change later to be included in the free version, but we want to give to the people who are enabling us to keep developing first.

Also the goal is to have great profiles here, so that people can start importing the apps they want from the start 😁 which also makes the global export less important.

**Reply by Marc05** [reply]
I think that configuration backup and restore, including the convenience of a single/easy backup option, is basic functionality that should be part of the free version. I do see convenience features such as sync, config history, native integration with third-party storage (e.g. Nextcloud), etc. as additional functionality that could be placed behind a subscription. Devs need to be paid after all - Portmaster is awesome! I just kindly ask that an easy-button config backup is left to be free for everyone. It would lessen the friction to sharing app profiles with the community as well; at least that's part of why I've yet to share any of my profiles.

**Comment by CCS86** [comment]
Any updates on this?

**Reply by NL-TCH** [reply]
It is sad to see this project which had a very active community dying or already being dead. So don't count on any features being added at all. Very curious how the original creators Raphael and David look back on their sale of the org. (ofc not blaming them, they were great guys!!)
